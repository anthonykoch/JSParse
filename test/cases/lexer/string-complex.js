"hello \r we \n are \\r \\n \\\r \\\n \\\r\\\n 👍 \'"
'hello \r we \n are \\r \\n \\\r \\\n \\\r\\\n 👍 \"'
