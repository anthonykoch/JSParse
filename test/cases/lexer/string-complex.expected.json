---
name: String - complex
description: A string with all the fixins
---

[
  {"type": 1, "value": "\"hello \\r we \\n are \\\\r \\\\n \\\\\\r \\\\\\n \\\\\\r\\\\\\n 👍 \\'\"", "line": 1, "column": 0, "start": 0, "end": 53 },
  {"type": 1, "value": "'hello \\r we \\n are \\\\r \\\\n \\\\\\r \\\\\\n \\\\\\r\\\\\\n 👍 \\\"'", "line": 2, "column": 0, "start": 54, "end": 107 }
]
